backend/tests/test_offline_patch.py

```
@@ -0,0 +1,113 @@
+    )
+    assert result is sentinel
+
+
+def test_passthrough_on_success(monkeypatch):
+    """When model_info returns non-Mistral tags the original falls through and returns the tokenizer unchanged."""
+    _apply_patch()
+
+    import huggingface_hub
+
+    class FakeInfo:
+        tags = ["model-type:qwen", "language:en"]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)